azure-client-runtime/src/main/java/com/microsoft/azure/AzureServiceClient.java

```diff
@@ -49,6 +56,20 @@ protected AzureServiceClient(String baseUrl, ServiceClientCredentials credential
      */
     protected AzureServiceClient(RestClient restClient) {
         super(restClient);
+        rpHttpClient = new RxNettyClient(new ChannelHandlerConfig(new Func0<ChannelHandler>() {
+            @Override
+            public ChannelHandler call() {
+                return new AzureTokenCredentialsHandler((AzureTokenCredentials) AzureServiceClient.this.restClient().credentials());
+            }
+        }, true));
+    }
+
+    private final HttpClient rpHttpClient;
```

Comment:
I know that "rp" stands for RestProxy (it does, right?), but no one else will. Just call it "httpClient" for now.

Reply:
Naming conflict with a base class member-- perhaps I'll rename the original `httpClient` to `okHttpClient` and this to merely `httpClient`?